src/app.css

```diff
@@ -49,3 +49,8 @@ input[type='number'] {
 input[type='radio'] {
   @apply accent-purple-500;
 }
+
+/* noUiSlider "Slider" component */
+.noUi-connect {
+  background: rgb(99 5 240 / 1) !important;  /* bg-purple-500 */
+}
```

Comment:
Would this still work if we put it in a `style` tag in the Slider component? Would be great to keep out of the global CSS if possible.

Reply:
didn't work unfortunately - tried a bunch of things!